services/pcs/src/pcs/cli.py

```diff
@@ -28,3 +28,9 @@
 def sync_run_rest_app():
     """Run the HTTP REST API."""
     asyncio.run(run_rest_app())
+
+
+@cli.command(name="publish-events")
+def sync_run_publish_events(all: bool = False):
+    """Publish pending events. Use `--all` to (re)publish all events regardless of status."""
```

Comment:
```suggestion
def sync_run_publish_events(all: Annotated[bool, typer.Argument(help="to (re)publish all events regardless of status")] = False):
    """Publish pending events."""
```

Maybe this is even better.